tests/test_D2_model_constraints.py

```diff
@@ -194,9 +194,13 @@ def run_parts(margs):
 
         self.dict_values, self.model, self.dict_model = run_parts()
         self.exp_emission_limit = 1000
+        self.min_renewable_share = 0.60
```

Comment:
Would rename to `exp_min_renewable_share` as you use it below as expected value in the assertion statement.